runtime/src/commonMain/kotlin/pbandk/wkt/descriptor.kt

```diff
@@ -400,6 +400,7 @@ data class FieldDescriptorProto(
     val oneofIndex: Int? = null,
     val jsonName: String? = null,
     val options: pbandk.wkt.FieldOptions? = null,
+    val proto3Optional: Boolean? = null,
```

Comment:
I think you accidentally committed a version of the proto files generated with a newer version of protobuf. That's why CI is failing. CI is pinned to protobuf 3.10.1 right now.